app/javascript/components/admin/site_settings/appearance/BrandColor.jsx

```diff
@@ -1,15 +1,38 @@
 import React from 'react';
+import { Stack } from 'react-bootstrap';
 import useSiteSettings from '../../../../hooks/queries/admin/site_settings/useSiteSettings';
-import BrandColorForm from './forms/BrandColorForm';
+import BrandColorPopover from './BrandColorPopover';
 
 export default function BrandColor() {
   const { data: siteSettings } = useSiteSettings();
 
+  const brandDefaultColors = [
+    '#B80000', '#DB3E00', '#FCCB00', '#008B02', '#006B76', '#1273DE', '#004DCF', '#5300EB',
+  ];
+
+  const brandLightDefaultColors = [
+    '#EB9694', '#FAD0C3', '#FEF3BD', '#C1E1C5', '#BEDADC', '#C4DEF6', '#BED3F3', '#D4C4FB',
+  ];
+
```

Comment:
I think we should drop the default colors - they aren't useful for users

Reply:
Would you rather have only an input for the hex value?
IMO this is better than the saturation/color field.
I think it looks good and do no harm.

Reply:
I could do a full custom block without the default colors.
![image](https://user-images.githubusercontent.com/43917914/181575762-271d0a32-5a90-408a-88ad-f01353a5caa0.png)
I'm not sure this will be ready for the demo tomorrow tho.